package.json

```diff
@@ -50,7 +45,10 @@
 					"*.ts"
 				],
 				"rules": {
-					"@typescript-eslint/no-unnecessary-condition": "error",
+					"ava/assertion-arguments": "off",
```

Comment:
`strip-indent` is not ESM and the whole TS/ESM setup makes it impossible to get its types without using `allowSyntheticDefaultImports` in `tsconfig`, which can create a broken ESM module.

Reply:
It is now https://github.com/sindresorhus/strip-indent/releases/tag/v4.0.0